src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -12,6 +12,7 @@ export enum CorpTypeCd {
   BC_COMPANY = 'BC',
   BENEFIT_COMPANY = 'BEN',
   CONTINUE_IN = 'C',
+  BEN_CONTINUE_IN = 'CBEN',
```

Comment:
This should have a corresponding record in `CorpInfoArray` below.

Reply:
Thanks for catching that Sev.